routes/api_v2.php

```diff
@@ -741,3 +742,5 @@ function () {
 });
 
 Route::get('/type-entity', EntityTypeController::class);
+
+Route::post('/locale/{locale}', UserLocaleController::class);
```

Comment:
I already approved the FE PR, but I think this needs a change. A REST structure like this would indicate to me that the FE is trying to create a Locale, which is weird. Also, for a Post, I would expect the value to be in the body, not the URL. We're going to be doing a lot of work to make our API actually RESTful in the v3 architecture, but the fewer missteps we have in new APIs in v2, the easier that will be.

I think this should instead be structured as something more like 
```
PATCH /api/v2/user
```
or 
``` 
PATCH /api/v2/user/locale
```
with the locale value in the body. If we already had a proper user update endpoint, I'd say this should just be part of that patch, but for instance password reset is done through `PATCH /api/v2/user/change`, which is also very weird. Given that, I think `PATCH /api/v2/user/locale` is (while still a bit strange), the less confusing option above given the current state of the PHP API.

Ideally (in v3), we'll have a simple patch user endpoint (likely something like `PATCH /user-service/user/me` with `me` being a "magic" ID so we don't have to pass around the current user's ID in the API) that can take payloads with various things that can be updated (like password and locale) and then returns the full updated Me user resource as the response.